nucleus/cluster/admin/src/main/java/fish/payara/admin/cluster/StopDeploymentGroupCommand.java

```diff
@@ -120,14 +119,19 @@ public void execute(AdminCommandContext context) {
         ClusterCommandHelper clusterHelper = new ClusterCommandHelper(domain,
                 runner);
 
-        ParameterMap map = null;
+        ParameterMap map = new ParameterMap();
         if (kill) {
-            map = new ParameterMap();
             map.add("kill", "true");
         }
+        if (instanceTimeout > 0) {
```

Comment:
So it's ignored if it's not greater than 0?
Commands shouldn't just ignore incorrect parameters, they should fail.